code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -23,7 +24,7 @@
 	if(!istype(C))
 		return
 
-	if((CLUMSY in user.mutations) && prob(50) && !istype(src, /obj/item/restraints/handcuffs/toy))
+	if((CLUMSY in user.mutations) && prob(50) && (ignoresClumsy == FALSE))

```

Comment:
you can replace with if((CLUMSY in user.mutations) && prob(50) && (!ignoresClumsy))

!ignoresClumsy does the same as the FALSE check

Reply:
You're right as well.

Ite one sec